test/IntegrationTests/_bcl/EncryptionTestsV1InteropV1N.cs

```diff
@@ -92,49 +92,37 @@ public EncryptionTestsV1InteropV1N() : base(KmsKeyIdProvider.Instance)
 
             s3EncryptionClientMetadataModeAsymmetricWrapV1 =
                 new Amazon.S3.Encryption.AmazonS3EncryptionClient(asymmetricEncryptionMaterialsV1);
-            RetryUtilities.ForceConfigureClient(s3EncryptionClientMetadataModeAsymmetricWrapV1);
```

Comment:
is `ForceConfigureClient` still used or could the entire method also be removed?

Reply:
The whole file containing RetryUtilities is deleted. Nothing left behind.